app/lib/clients/vault/policy.rb

```diff
@@ -1,12 +1,23 @@
 module Clients
   class Vault
     module Policy
+      extend Entity
+
       def rotate_token
         create_astral_policy
         token = create_astral_token
         Clients::Vault.token = token
       end
 
+      def assign_policy(identity, policy_path)
```

Comment:
Nit:  I prefer "policy_name" to "policy_path".  That is the terminology used in the vault docs.

Reply:
done